### DISCUSSION TITLE
Questions on WARNINGs during METplus beta testing

### DISCUSSION THREAD

**Discussion by PerryShafran-NOAA** [dtcenter/METplus#2521] [Use Cases]
We had several WARNINGs pop up when running an EVS-wide test on the new beta versions of the next METplus coordinated release.  Most seem to do with deprecated variables or the changes in variable names.  Can you let us know what the issue may be with these?

1. WARNING: PB2NC_SKIP_TIMES is deprecated. Please use PB2NC_SKIP_VALID_TIMES.  

2. WARNING: Both POINT_STAT_CLIMO_MEAN_INPUT_TEMPLATE and POINT_STAT_CLIMO_MEAN_FILE_NAME are set. Using value set in POINT_STAT_CLIMO_MEAN_FILE_NAME (/lfs/h2/emc/vpppg/noscrub/emc.vpppg/EVS/fix/climos/atmos/era5/mean_{valid?fmt=%m%d})  

3. WARNING: Both POINT_STAT_CLIMO_STDEV_INPUT_TEMPLATE and POINT_STAT_CLIMO_STDEV_FILE_NAME are set. Using value set in POINT_STAT_CLIMO_STDEV_FILE_NAME (/lfs/h2/
emc/vpppg/noscrub/emc.vpppg/EVS/fix/climos/atmos/era5/stdev_{valid?fmt=%m%d}).

4. WARNING: Both GRID_STAT_CLIMO_MEAN_INPUT_TEMPLATE and GRID_STAT_CLIMO_MEAN_FILE_NAME are set. Using value set in GRID_STAT_CLIMO_MEAN_FILE_NAME (/lfs/h2/emc/vpppg/noscrub/emc.vpppg/EVS/fix/climos/atmos/ccpa/ccpa_24hr_1p0_prob_{valid?fmt=%m%d}.grib2).  

Thanks!

Perry

**Comment by georgemccabe** [comment]
For the 1st warning, you can change all instances of `PB2NC_SKIP_TIMES` to `PB2NC_SKIP_VALID_TIMES` to get rid of the warning. The use case should run as expected despite this warning being displayed. The wrappers now support skipping times based in initialization times as well, so the config variables were renamed to be more explicit.

For the next 3 warnings, I would compare the values set for the two variables listed in each warning. If they differ, determine which value is correct, set that value for the `*_FILE_NAME` variable, and remove the corresponding `*_INPUT_TEMPLATE` variable.

I ran some tests and discovered that the last 3 warnings may appear even if the `*_FILE_NAME` variables are not set due to the bug described in #2520
